### ISSUE TITLE
Accessing item parameter from Dash callbacks

### ISSUE THREAD

**Issue by CY-ipercept** [RenaudLN/dash-pydantic-form#7]
Thank you for this amazing tool! I've been trying to integrate it into existing projects and I have a question about accessing the `item` parameter. Is it possible to retrieve the model class that the form is built upon, similar to how we retrieve the data that the form contains?

```
if value:
    return ModelForm(
        item=value,
        aio_id="form",
        form_id=id_map.get(value),
    )

@callback(
    Output(ModelForm.ids.errors("MATCH", "MATCH"), "data"),
    Input(ModelForm.ids.main("MATCH", "MATCH"), "data"),
    Input(ModelForm.ids.main("MATCH", "MATCH"), "item"),
    prevent_initial_call=True,
)
```

I would like to render different forms based on something else that the user chose, hence I need different models and I need to call the `model_validate` function

**Comment by RenaudLN** [comment]
Hey @CY-ipercept yes, there is a way. It's not possible to store the class itself but the name of the Model is stored with the form and a util function allows you to retrieve the model class from it:

```py
from dash_pydantic_form.utils import get_model_cls

@callback(
    Output(ModelForm.ids.errors("MATCH", "MATCH"), "data"),
    Input(ModelForm.ids.main("MATCH", "MATCH"), "data"),
    State(ModelForm.ids.model_store("MATCH", "MATCH"), "data"),
    prevent_initial_call=True,
)
def handle_form(form_data: dict, model_name: str):
    model_cls = get_model_cls(model_name)
    model_cls.validate(form_data)
    # ...
```

**Comment by CY-ipercept** [comment]
Thank you, that helps a lot! Will you also be adding more features? I have a Pydantic class where one or more of the fields have the type of Dict[str, AnotherModel] or Dict[str, int]. I've been custom components to help me handle such cases but it would be really helpful if such functionality can be built into the library as well!

**Comment by RenaudLN** [comment]
Yes I will keep developing this library more. Interesting use case with the dicts of things, I'll think about what that could look like in terms of UI and let you know here if this progresses 🙂 

In the meantime I'll close this issue, feel free to open one for the dict feature request.

**Comment by RenaudLN** [comment]
> Thank you @RenaudLN , that helps a lot! Will you also be adding more features? I have a Pydantic class where one or more of the fields have the type of `Dict[str, AnotherModel]` or `Dict[str, int]`. I've been custom components to help me handle such cases but it would be really helpful if such functionality can be built into the library as well!

Hey @CY-ipercept, dicts are now available in 0.2.1, see the doc here: https://pydf-docs.onrender.com/api/dict_field

**Comment by CY-ipercept** [comment]
Thank you for the help! @RenaudLN
